parlai/mturk/core/dev/react_server/server/server.js

```diff
@@ -208,12 +642,13 @@ app.post('/sns_posts', async function(req, res, next) {
         receiver_id: world_id,
         data: data,
       };
-      _send_message(world_id, 'new packet', msg);
+      _send_message(world_id, SNS_MESSAGE, msg);
```

Comment:
where is this `world_id` coming from?

Reply:
`world_id` is a global that is set up when ParlAI first makes contact with this server.